nuxeo-ai-web-ui/src/main/resources/web/nuxeo.war/ui/i18n/messages.json

```diff
@@ -0,0 +1,19 @@
+{
+  "admin.ai.heading": "Artificial Intelligence Admin",
+  "app.title.admin.aicore": "Intelligence",
+  "admin.ai.export": "Dataset Export",
+  "admin.ai.stats": "Dataset Statistics",
+  "admin.ai.intel": "Intelligence",
+  "admin.ai.inputs" : "Input properties",
+  "admin.ai.outputs" : "Output properties",
+  "admin.ai.split" : "Split",
+  "admin.ai.stats.action" : "Get Statistics",
+  "admin.ai.stats.none" : "The query matches no results",
+  "admin.ai.stats.error" : "Failed to get the Dataset stats",
+  "admin.ai.export.action" : "Export Dataset",
+  "admin.ai.export.success": "Export Dataset Action Started",
+  "admin.ai.export.error": "Error exporting the Dataset",
+  "admin.ai.stats.field" : "Property Name",
+  "admin.ai.stats.type" : "Statistic Type",
+  "admin.ai.stats.value" : "Value"
+}
```

Comment:
Just for reference: https://doc.nuxeo.com/corg/user-interface-guidelines/
There are some missing details on message format in order to be coherent with the other pages and product guidelines.